recipe/meta.yaml

```diff
@@ -7,10 +7,12 @@ package:
 source:
   fn: mpld3-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/m/mpld3/mpld3-{{ version }}.tar.gz
-  sha256: b080f3535238a71024c0158280ab4f6091717c45347c41c907012f8dd6da1bd5
+  sha256: c589db8b661aee25c93e198e2e18ed47b9a96951de41d96241345acec5f819ab
 
 build:
   number: 0
+  skip: true  # [py<36]
```

Comment:
`noarch: python` packages should not have skip blocks based on python versions.

Reply:
Per your request the `skip` section was removed:
https://github.com/AnacondaRecipes/mpld3-feedstock/blob/update_to_0.5.7/recipe/meta.yaml#L13&&#L14